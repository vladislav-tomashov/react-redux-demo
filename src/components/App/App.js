import React from "react";
// import logo from "../../logo.svg";
import LoadButton from "../LoadButton/LoadButton";
import Spinner from "../Spinner/Spinner";
import CurrencyRates from "../CurrencyRates/CurrencyRates";

import "./App.css";

const App = () => (
  <React.Fragment>
    <Spinner />
    <div className="App">
      <header className="App-header">
        {/* <img src={logo} className="App-logo" alt="logo" /> */}
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
        <p>
          <LoadButton />
        </p>
        <CurrencyRates />
      </header>
    </div>
  </React.Fragment>
);

export { App as default };
