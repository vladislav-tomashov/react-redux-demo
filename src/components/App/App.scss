@import "../../styles/settings";

.App {
  text-align: center;
  background-color: $black;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  color: white;
  padding-bottom: $l-size;
}
