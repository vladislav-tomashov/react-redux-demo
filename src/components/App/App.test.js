import React from "react";
import { shallow } from "enzyme";
import App from "./App";

describe("App component", () => {
  test("renders without crashing", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
