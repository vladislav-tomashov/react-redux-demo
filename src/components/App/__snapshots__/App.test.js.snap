// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders without crashing 1`] = `
<Fragment>
  <Connect(Spinner) />
  <Connect(ErrorModal) />
  <div
    className="App"
  >
    <p>
      <Connect(LoadButton) />
    </p>
    <Connect(CurrencyRatesList) />
  </div>
</Fragment>
`;
